src/shared/api/api.test.ts

```diff
@@ -410,49 +445,25 @@ describe('when using a graphql mutation', () => {
     })
   })
 
-  describe('when the mutation supports serviceless', () => {
+  describe('when the mutation supports service-less', () => {
     it('resolves with the data', async () => {
+      setup()
       const result = await Api.graphqlMutation({
-        query: 'query MyInfo { me }',
-        mutationPath: 'me',
+        query: 'mutation CreateToken { createApiToken { error token } }',
+        mutationPath: 'createApiToken',
         supportsServiceless: true,
       })
 
       await waitFor(() =>
         expect(result).toEqual({
           data: {
-            me: 'Codecov',
+            createApiToken: {
+              error: null,
+              token: 123,
+            },
           },
         })
       )
     })
   })
-
-  describe('when graphql query returns a 401 error', () => {
```

Comment:
do we no longer need this test?

Reply:
Ahh, if you read a couple lines below, it's actually checking for a 403 error. I renamed, and moved this test to a more suitable location.

Reply:
Line 230 on the right.

Reply:
ah gotcha - thanks sounds good!!